.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: swift
+os: osx
+osx_image: xcode9
+sudo: required
+env: SWIFT_SNAPSHOT=4.0
+script: swift test
+
```

Comment:
Fun fact - when I worked on tic tac toe in Swift 3 when I was an apprentice, I spent days (literally) trying to get Travis CI working with xcode/swift/etc. After all that time, my mentors just said forget it because none of us could get it to work. I am really happy that the Swift ecosystem has matured to the point where Travis CI integration is straightforward.

Reply:
I wouldn't say straightforward. Did you see 3-4 commits that has something to do with travis.yml all resulted in failed CI build? The Travis documentation was not very helpful. Luckily I found a repo where I can tweak their travis.yml file to work with my stuff.